controls/nginx_spec.rb

```diff
@@ -190,7 +190,7 @@
   title 'Control simultaneous connections'
   desc 'NginxHttpLimitZone module to limit the number of simultaneous connections for the assigned session or as a special case, from one IP address.'
   describe parse_config(nginx_parsed_config, options) do
-    its('limit_conn_zone') { should eq '$binary_remote_addr zone=default:10m' }
+    its('limit_conn_zone') { should eq '$binary_remote_addr zone=value:10m' }
```

Comment:
That one is not an Inspec variable but an nginx configuration. I think it should stay to `default`.